### PR TITLE
Document edits not changing the semantic

### PR DESCRIPTION
**Pull request by mambelli**
Some edits to improve the grammar and punctuation, to use the USA English spelling to ease spell-checking (both UK and USA variants were present and some cited documents and technical terms use the USA spelling), and to clarify some sentences (minimal changes). Changes were kept to the minimum with the goal of not changing the semantics and not adding or removing content.

### CODE REVIEW

File: profile.md [hunk 1]
```diff
@@ -111,23 +111,23 @@ One item not addressed in detail in this document is how the issuer decides on w
...
    </td>
-   <td>VO https url
+   <td>VO https URL
    </td>
   </tr>
   <tr>
    <td><strong>Authorization Server</strong>
    </td>
-   <td>The entity which produces (“issues”) the token. For WLCG authorization, this is a service run by the VO that is asserting the identity or the authorization to access the VO’s resources. This term is defined by OAuth2 and may be referred to as the Authorization Server.
+   <td>The entity that produces (“issues”) the token. For WLCG authorization, this is a service run by the VO that is asserting the identity or the authorization to access the VO’s resources. This term is defined by OAuth2, sometimes referred also as Authorization Provider. Is equivalent to an issuer in OIDC terminology.
```
Comment (DrDaveD) [inline profile.md:122]: "referred also as" doesn't make for a good sentence.  I suggest "referred to as".

File: profile.md [hunk 2]
```diff
@@ -286,7 +286,7 @@ Usage of this claim is OPTIONAL. However, the <strong><code>wlcg.groups</code></
...
+   <td>The <strong><code>aud</code></strong> claim represents the audience or audiences the token is intended for. In the general case, the <strong><code>aud</code></strong> value is an array of case-sensitive strings. As specified in [RFC 7519 section 4.1.3](https://tools.ietf.org/html/rfc7519#section-4.1.3), in the common special case when there is one audience, the aud value MAY be a single case-sensitive string. The special string value of “https://wlcg.cern.ch/jwt/v1/any” signifies that the issuer intends the token to be valid for all relying parties.  See the discussion below for further guidance on picking values for the `aud` claim.
```
Comment (DrDaveD) [inline profile.md:289]: This link doesn't render properly under "view file".  I suppose it needs the `<a>` html format.

**Review by DrDaveD — CHANGES_REQUESTED** [review]

File: profile.md [hunk 3]
```diff
@@ -37,23 +37,23 @@ _Authored by the WLCG AuthZ Working Group_
...
+The trust model used in this profile is VO-centric and uses the concept of transitive trust: an individual establishes an identity within the VO (through an identity-proofing mechanism not described here) and any authentication mechanism happens within the scope of the VO.  This is in strong contrast to the current X.509-based system where a global identity is established completely orthogonal to the VO.
 
-The WLCG has identified two strong use cases for these profiles: issuing information about an identity and issuing bearer-token-based authorizations.  Identities are typically needed within a VO’s services, which might provide different views or authorization based on the individual’s identity within the VO.  
+The WLCG has identified two strong use cases for these profiles: issuing information about an identity and issuing bearer-token-based authorizations.  Identities are typically needed within a VO’s service, which might provide different views or authorization based on the individual’s identity within the VO.
```
[the remark this answers is not in the document]
Comment (mambelli) [reply profile.md:48]: Then maybe "within VO's services, which":  without the "a" is better for the plural services and "VO's" says that we refer to one VO.

File: profile.md (hunk 1 above)
Comment (mambelli) [reply profile.md:122]: done

File: profile.md [hunk 4]
```diff
@@ -79,15 +79,15 @@ One item not addressed in detail in this document is how the issuer decides on w
...
+   <td>OpenID Connect (OIDC) is an authentication layer on top of OAuth 2.0. It leverages a specific OAuth 2.0 flow to provide authentication information and relevant identity attributes. OIDC flows may issue OAuth2 Access Tokens, Refresh Tokens, as well as ID Tokens.
    </td>
    <td>
    </td>
   </tr>
   <tr>
    <td><strong>Token</strong>
    </td>
-   <td>JSON Web Token (JWT).  A string representing a set of claims (pieces of information about a subject) as a JSON object that is encoded in a JWS or JWE, enabling the claims to be digitally signed or MACed and/or encrypted. <a href="http://tools.ietf.org/html/rfc7519">JSON Web Token</a> - RFC 7519
+   <td>JSON Web Token (JWT).  A string representing a set of claims (pieces of information about a subject) as a JSON object that is encoded in a JSON Web Signature (<a href="http://tools.ietf.org/html/rfc7515">JWS, RFC 7515</a>) or JSON Web Encryption (<a href="http://tools.ietf.org/html/rfc7516">JWE, RFC 7516</a>), enabling the claims to be digitally signed or MACed and/or encrypted. <a href="http://tools.ietf.org/html/rfc7519">JSON Web Token - RFC 7519</a>.
```
[the remark this answers is not in the document]
Comment (mambelli) [reply profile.md:90]: done

File: profile.md (hunk 2 above)
Comment (mambelli) [reply profile.md:289]: probably because of the table. fixed

**Comment by mambelli** [comment]
@DrDaveD I think I addressed all the comments

File: profile.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (maarten-litmaath) [reply profile.md]: The latter suggestion is incorrect English.  The original text is still the best.

**Comment by maarten-litmaath** [comment]
Ciao Marco,
thanks for your many suggestions, many of which make sense indeed.

They will have to be applied selectively, though, because some are contentious and would encounter opposition.

For example, I am not a fan of replacing `RFC number` with `RFCnumber`, because I find the original much easier to read and also the official RFC docs have a space preceding the number.  For example:
https://datatracker.ietf.org/doc/html/rfc7643

**Comment by mambelli** [comment]
> They will have to be applied selectively, though, because some are contentious and would encounter opposition.

Let me know which can work and I can remove or separate the others

> For example, I am not a fan of replacing `RFC number` with `RFCnumber`, because I find the original much easier to read and also the official RFC docs have a space preceding the number. For example: https://datatracker.ietf.org/doc/html/rfc7643

I prefer the space as well but most RFCa seem not to have a space, especially when the RFCs are referenced, probably because they are treated as citations "[RFCnnn]".
Once there is consensus, I can change the patch to have one format for the whole document.

File: profile.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (DrDaveD) [reply profile.md]: Each VO has multiple services, so I agree with Maarten that the original text is the best here.

**Comment by DrDaveD** [comment]
I'm OK with the space in `RFC NNN`, as long as it's consistent.

**Comment by maarten-litmaath** [comment]
Superseded by #51 .

**Pull request closed without merging**